cmd/csi-resizer/main.go

```diff
@@ -86,24 +97,42 @@ func main() {
 	}
 
 	resizerName := csiResizer.Name()
+	rc := controller.NewResizeController(resizerName, csiResizer, kubeClient, *resyncPeriod, informerFactory)
+	run := func(ctx context.Context) {
+		informerFactory.Start(wait.NeverStop)
+		rc.Run(*workers, ctx)
 
-	var leaderElectionConfig *util.LeaderElectionConfig
-	if *enableLeaderElection {
-		if leaderElectionIdentity == nil || *leaderElectionIdentity == "" {
-			klog.Fatal("--leader-election-identity must not be empty")
+	}
+
+	if !*enableLeaderElection {
+		run(context.TODO())
+	} else {
+		if *leaderElectionIdentity == "" {
```

Comment:
I would like to get rid of this flag since csi-lib-utils will check the pod hostname if the identity is not explicitly set.